modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,80 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
+
+It is possible to create read privileges that are based on properties of nodes.
+Each property-based privilege can only be restricted by a single property.
+To specify the property/value conditions of the read privilege the `pattern` syntax described below is used,
+for more information about read privilege syntax see xref:authentication-authorization/privileges-reads.adoc[read privilege] page.
+
+Adding property-based access control may lead to a significant performance overhead in certain scenarios.
+See xref:authentication-authorization/limitations.adoc#property-based-access-control-limitations[Limitations] for more detailed information.
+To reduce the performance impact, it is recommended to use the Block Storage format as it is better optimized for the kind of read required for the resolution of property-based privileges.
+
+Some of the factors that can worsen the impact on performance when having property rules are:
+
+* The number of properties on the nodes concerned (more properties = greater performance impact)
+* The number of property-based privileges (more property-based privileges = greater performance impact).
+* The type of the privilege: `TRAVERSE` property-based privileges have a greater performance impact than `READ` property-based privileges.
+* The type of storage medium in operation. The performance impact of property-based privileges will be considerably amplified by accessing disc storage.
+
+For performance-critical scenarios, it is recommended to design privileges based on labels.
+For more information, see xref:authentication-authorization/privileges-reads.adoc[Read privileges].
+
+Pattern syntax:
+[source, syntax, role="noheader"]
+----
+([var][:label["|" ...]] "{" property: value "}")
+| (var[:label["|" ...]]) WHERE [NOT] var.property { { = | <> | > | >= | < | <= } value | IS NULL | IS NOT NULL }
+| (var[:label["|" ...]] WHERE [NOT] var.property { { = | <> | > | >= | < | <= } value | IS NULL | IS NOT NULL } )
```

Comment:
one thing to keep in mind for when this is ready for merge/GA is that we'll need to update the `additions, deprecations, removals` section (currently in cypher manual, but I think there was some talk about making something for the operations manual/administration section as well since it moved manual so I guess wherever it is at time of GA)